libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/MatrixArea/MatrixAreaUtils.ts

```diff
@@ -152,14 +138,14 @@ export function createCohortStatsFromSelectedCells(
 
 export function createCompositeFilterFromCells(
   selectedCells: boolean[],
-  jsonMatrix: { matrix: any[]; category1: any; category2: any },
-  selectedFeature1: string,
-  selectedFeature2: string,
+  jsonMatrix: IErrorAnalysisMatrix,
+  selectedFeature1: string | undefined,
+  selectedFeature2: string | undefined,
   baseCohort: ErrorCohort,
   features: string[]
 ): ICompositeFilter[] {
   const feature2IsSelected =
-    selectedFeature2 !== noFeature && selectedFeature2 !== selectedFeature1;
+    selectedFeature2 && selectedFeature2 !== selectedFeature1;
```

Comment:
the user can actually select noFeature and user should only see one, I don't see where this check was moved to, but this would be a bug if the check was removed altogether